app/src/forms/email/reminderService.js

```diff
@@ -0,0 +1,296 @@
+
+const { getAvailableDates } = require('../common/utils');
+const emailService  = require('./emailService');
+const moment = require('moment');
+const {  EmailTypes, ScheduleType } = require('../common/constants');
+const config = require('config');
+const log = require('../../components/log')(module.filename);
+const {  SubmissionData, UserFormAccess, Form } = require('../common/models');
+const { Roles } = require('../common/constants');
+
+const service = {
+  _init: async ()=> {
+    const forms = await service._getForms();
+
+    const q     = await service._getReminders(forms);
+
+    const referer = service._getReferer();
+    const resolve = [];
+    const errors = [];
+    let mail = 0;
+    
+    if(q.length!==0) {
+      for(let i = 0 ; i < q.length ; i++) {
+        if(!q[i].error){
+          const obj = await  service._runQueries(q[i].statement);
+          let result = await service._initStatement(obj);
+          let chesResponses = service._initMaillSender(result, referer);
```

Comment:
This should be `service._initMailSender(result, referer);`

Reply:
Solve